lib/plugins/dot-net.js

```diff
@@ -0,0 +1,37 @@
+import insertLink from '../insert-link';
+import nugetResolver from '../resolver/nuget';
+import { xmlRegExKeyValue } from './helper/regex-builder';
+
+function linkDependency(blob, key, value) {
+  const regex = xmlRegExKeyValue(key, value);
+
+  insertLink(blob.el, regex, {
+    pluginName: 'DotNet',
+    target: '$1',
+  });
+}
+
+export default {
+  name: 'DotNet',
+
+  resolve({ target }) {
+    return nugetResolver({ target });
+  },
+
+  getPattern() {
+    return {
+      pathPatterns: ['/packages.config$'],
+      githubClasses: [],
+    };
+  },
+
+  parseBlob(blob) {
+    const packageRegex = new RegExp('<\\s*package\\s.*id="(.*?)".*\\/>', 'g');
```

Comment:
Could you move this RegExp into [helper-grammar-regex-collection](https://github.com/OctoLinker/browser-extension/blob/02c0d0654a1436cdc90da33c92f82eb05463e8c2/packages/helper-grammar-regex-collection/index.js) and add a few [tests](https://github.com/OctoLinker/browser-extension/blob/02c0d0654a1436cdc90da33c92f82eb05463e8c2/packages/helper-grammar-regex-collection/test.js) of it? (commit a3ec0f9c048873509b6c60bd05e0019d2cf32657 has an example of adding/testing a new regex)